march_joint_inertia_controller/src/inertia_estimator.cpp

```diff
@@ -0,0 +1,290 @@
+// Copyright 2020 Project March.
+#include <hardware_interface/joint_command_interface.h>
+#include <hardware_interface/joint_state_interface.h>
+#include "march_joint_inertia_controller/inertia_estimator.h"
+#include "math.h"
```

Comment:
`math.h` is an old deprecated header, you should use `cmath`
```suggestion
#include <cmath>
```